CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # New Relic Ruby Agent Release Notes #
 
+  ## v8.6.0
+
+  * **Telemetry-in-Context: a quick way to view logs no matter where you are in the platform**
```

Comment:
I think we have to change the title slightly, even though the google doc draft shows something different. this was decided in a Slack thread: 

```suggestion
* **Telemetry-in-Context: Automatic Application Logs, a quick way to view logs no matter where you are in the platform**
```

Reply:
Thanks for letting me know! I was going off the google doc, so I didn't see this